packages/webhooks/schema/index.js

```diff
@@ -8,6 +13,19 @@ function getWebhookModels (schemaPath) {
     }
 }
 
+function getWebhookTasks (taskPriority = DEFAULT_QUEUE_NAME) {
+    if (WEBHOOK_TASKS.size === 0) {
```

Comment:
It's better to check names here `if (!WEBHOOK_TASKS.has(<task-name>)) // add task `